### PR TITLE
Add geopoint accuracy

### PR DESCRIPTION
**Pull request by joeferner**
This is the first step which does not require re-indexing geopoint objects. The next step will be to re-index geopoint objects to geo-circles.

### CODE REVIEW

File: core/src/main/java/org/vertexium/type/GeoPoint.java [hunk 1]
```diff
@@ -92,6 +118,7 @@ public int hashCode() {
         hash = 47 * hash + (int) (Double.doubleToLongBits(this.latitude) ^ (Double.doubleToLongBits(this.latitude) >>> 32));
         hash = 47 * hash + (int) (Double.doubleToLongBits(this.longitude) ^ (Double.doubleToLongBits(this.longitude) >>> 32));
         hash = 47 * hash + (this.altitude != null ? this.altitude.hashCode() : 0);
+        hash = this.accuracy != null ? (47 * hash + this.accuracy.hashCode()) : 0;
```
Comment (mwizeman) [inline core/src/main/java/org/vertexium/type/GeoPoint.java]: This looks like a typo. If no accuracy is provided, the hash will always be 0

Comment (joeferner) [reply core/src/main/java/org/vertexium/type/GeoPoint.java]: Wow, how did you catch that?

Comment (mwizeman) [reply core/src/main/java/org/vertexium/type/GeoPoint.java]: Reading through the changes, it caught my attention that it didn't look like the other lines above it :-)

**Comment by joeferner** [comment]
This PR should be good to go now

**Pull request merged**